examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
<p>I wonder if it makes sense (or it is possible) to describe the different use case with LaTeX in mathematical expression?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>If you are referring to explaining the different sample weights use cases with math, I think that would be trick.</p><p>The formula to compute sample weight itself is simple, it either use another column as sample weight, or convert a binary column to sample weight by doing <code>tf.where(binary_column_value == 1, pos_weight, neg_weight)</code>. Not sure if that helps.</p><p>The rest of the logic of <code><a href="https://github.com/NVIDIA-Merlin/models/blob/f0bb535046a6de008f182caede2fd06341a26267/merlin/models/tf/outputs/block.py#L231-L285" target="_blank">ColumnBasedSampleWeight</a></code> is to deal with missing arguments and extracting the values for <code>weight_column_name</code> from features/target. </p><p> </p><p> </p>